zipline/algorithm.py

```diff
@@ -240,6 +241,9 @@ def __init__(self, *args, **kwargs):
                 commission=PerShare()
             )
 
+        # Cancel open orders at end of day by default
+        self.cancel_policy = EODCancel(warn_on_cancel=True)
```

Comment:
I don't think this default belongs in Zipline.  If we want to enforce this on Quantopian, we should move it into a downstream subclass.